README.md

```diff
@@ -1,4 +1,16 @@
-# Nva Datacite MDS
+# nva-doi-registrar-client
+
+[![Codacy Badge](https://app.codacy.com/project/badge/Grade/08aae785336d4c9f9c66ee3e5975889e)](https://www.codacy.com/gh/BIBSYSDEV/nva-doi-registrar-client/dashboard?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=BIBSYSDEV/nva-doi-registrar-client&amp;utm_campaign=Badge_Grade)
+
+Library and lambda functions which integrates towards Datacite (registry agency, RA).
+
+## API integrations
+
+Datacite MDS
+
+## Documentation about DOI
+
+See NVAs page in Confluence about [DOI](https://unit.atlassian.net/wiki/spaces/NVAP/pages/979763260/Digital+Object+Identifier+DOI)
```

Comment:
At the moment this documentation is in the WIKI, but this wiki is not open. We might want to move this documentation as a markdown file under `/docs` or something.